data/src/main/scala/ackcord/data/message.scala

```diff
@@ -66,6 +64,18 @@ object ImageData {
   }
 }
 
+sealed abstract class FormatType(val value: Int) extends IntEnumEntry
+object FormatType extends IntEnum[FormatType] with IntCirceEnumWithUnknown[FormatType] {
+  override def values: IndexedSeq[FormatType] = findValues
+
+  case object PNG extends FormatType(1)
+  case object APNG extends FormatType(2)
+  case object LOTTIE extends FormatType(3)
```

Comment:
I wasn't sure what case you wanted here, so I just did this because it looks nicer.